README.md

```diff
@@ -171,8 +171,8 @@ There are 2 approaches:
 - Using [AssertFunction()](https://pkg.go.dev/github.com/dop251/goja#AssertFunction):
 ```go
 const SCRIPT = `
-function f(param) {
-    return +param + 2;
+function sum(a, b) {
```

Comment:
This was changed by https://github.com/dop251/goja/pull/511 to make it more consistent with the following example. What I didn't realise is that the rest of the example was not changed. I think it's better to do that instead.

Reply:
Looks it #511 wrapped script body into a constant and adjusted the function body it self. Yet didn't sync the refer usage part.

I've turned both the examples to use the same more meaningful function `sum` (which might be more clear to grasp the logic) with the same notation and the same usage holder variables.